Sources/StreamChatUI/ChatMessageList/RETHINK/MessageThreadVC.swift

```diff
@@ -225,7 +157,10 @@ class MessageThreadVC<ExtraData: ExtraDataTypes>: _ViewController, UICollectionV
         let message = chatMessage(for: indexPath)
         
         let reuseId = cellReuseIdentifier(for: message)
-        let layoutOptions = cellLayoutOptionsForMessage(at: indexPath)
+        let isLastInGroup = isMessageLastInGroup(at: indexPath)
+        var layoutOptions = layoutOptionsResolver(message, isLastInGroup)
+        layoutOptions.remove(.threadInfo)
+        layoutOptions.remove(.continuousBubble)
```

Comment:
This will break grouping for threads which is not correct.

Reply:
fixed ✅